### PR TITLE
Updated adlib TypeScript Version

### PR DESCRIPTION
**Pull request by MikeTschudi**
- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

If changing an existing definition:
- [x] Include [tests for your changes](https://github.com/DefinitelyTyped/DefinitelyTyped#testing)

### CODE REVIEW

**Comment by danger-public** [comment]
Inspecting the JavaScript source for this package found some properties that are not in the .d.ts files.
The check for missing properties isn't always right, so take this list as advice, not a requirement.
## adlib ([<kbd>unpkg</kbd>](https://unpkg.com/browse/adlib@latest/))
was missing the following properties:
1. arborist
1. deepMapValues
1. getPropertyValue
1. getWithDefault
1. mapValues

<details>
<summary>as well as these 3 other properties...</summary>
<p>optionalTransform, pruneArray, pruneObject</p>
</details>

<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 98d5606593aba8ac91bbce3781d101b4354b73f8
</p>

**Review by MikeTschudi — COMMENTED** [review]
OK

File: types/adlib/index.d.ts [hunk 1]
```diff
@@ -3,7 +3,7 @@
 // Definitions by: Esri <https://github.com/Esri>
 //                 Mike Tschudi <https://github.com/MikeTschudi>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
-// TypeScript Version: 2.1
+// TypeScript Version: 3.9
```
Comment (elibarzilay) [inline types/adlib/index.d.ts:6]: Is there a 3.9 requirement?  (This is usually used to limit the TS versions, so if there's no newer feature you need it's best to leave it.)

Comment (MikeTschudi) [reply types/adlib/index.d.ts:6]: Thank you; there isn't. Let's cancel this, please.

**Pull request closed without merging**